DTAdmin/MainTabController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ViewController.swift
+//  DTAdmin
+//
+//  Created by ITA student on 10/10/17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import UIKit
+
+class MainTabController: UITabBarController, UITabBarControllerDelegate {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.delegate = self
+        
+        // Create Tab one
+        let tabOne = UINavigationController()
```

Comment:
Could you please rename it to something meaningful (e.g. groupTab instead of tabOne)? Please do the same for tabTwo below.